src/components/Renderer/widgets/WidgetMeta.tsx

```diff
@@ -17,8 +16,8 @@ export default function WidgetMeta({
 	schema = {},
 	uiSchema = {},
 }: WidgetMetaProps) {
-	const title = get(uiSchema, 'ui:title', schema.title || valueKey);
```

Comment:
I hope we were not exploiting empty string titles anywhere...

Reply:
Er... I believe we do. We explicitly set `ui:title` to null to prevent it displaying. This is important.

Reply:
Fwiw you could set it to `''` and it'd still work the same, but I'll keep the same behavior